test/testdata/valid_config.yaml

```diff
@@ -0,0 +1,38 @@
+Scicat:
+  Host: http://scicat:8080/api/v3
+  AccessToken: "token"
+Transfer:
+  Method: S3
+  S3:
+    Endpoint: s3:9000
+    Bucket: landingzone
+    Checksum: true
+    Location: "eu-west-1"
+    User: "minio_user"
+    Password: "minio_pass"
+Misc:
+  ConcurrencyLimit: 2
+  Port: 8888
+MetadataExtractors:
+  InstallationPath: ./parentPathToAllExtractors/
+  DownloadMissingExtractors: false
+  Extractors:
+  - Name: LS
+    GithubOrg: SwissOpenEM
+    GithubProject: LS_Metadata_reader
+    Version: v0.2.3
+    Executable: LS_Metadata_reader
+    Checksum: 8c5249c41a5b3464d183d063be7d96d9557dcb11c76598690f2c20bb06937fbe 
+    ChecksumAlg: sha256
+    CommandLineTemplate: "-i {{.SourceFolder}} -o {{.OutputFile}} {{.AdditionalParameters}}"
```

Comment:
@ytittes if you adapt the interface of the LS_Metadata_reader to such an interface, it should work and the extractor should be callable from the ingestor.
Each line in `AdditionalParameters` will be added in `{{.AdditionalParameters}}` with a space separating them